Xamarin/SSW.Consulting/SSW.Consulting.Android/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="7" android:versionName="1.5" package="com.ssw.consulting" android:installLocation="auto">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="8" android:versionName="1.5" package="com.ssw.consulting" android:installLocation="auto">
```

Comment:
Andreas - looks like the version here has not been updated

Reply:
Fixed